source/stylesheets/globals/imports/finescss/_fine_mixins.scss

```diff
@@ -18,6 +18,96 @@
   list-style: none;
 }
 
+// Create iOS style text underlines
```

Comment:
LDT (unless you want to) Just cause I'm a stickler for indentation, can we have the comments remain to the left and use spaces to show the correct indentation?